src/Components/AffectedClustersTable/AffectedClustersTable.js

```diff
@@ -134,10 +138,19 @@ const AffectedClustersTable = ({ query, rule, afterDisableFn }) => {
         return row?.cells[0].toLowerCase().includes(filters.text.toLowerCase());
       })
       .sort((a, b) => {
-        if (filters.sortDirection === 'asc') {
-          return a?.cells[0].localeCompare(b?.cells[0]);
+        let fst, snd;
+        const d = filters.sortDirection === 'asc' ? 1 : -1;
+        switch (filters.sortIndex) {
+          case AFFECTED_CLUSTERS_NAME_CELL:
+            if (filters.sortDirection === 'asc') {
+              return a?.cells[0].localeCompare(b?.cells[0]);
```

Comment:
This is basically the same code done for it to work. If there is an issue in that line, we will miss it. I would recommend another approach, sorting directly the data and comparing that with our result. What do you think?